source/_data/glossary.yml

```diff
@@ -31,7 +33,11 @@
 - topic: hass
   description: "HASS or [hass](/docs/tools/hass/) is often used as an abbreviation for Home Assistant. It is also the command-line tool."
 - topic: Hass.io
-  description: "[Hass.io](/hassio/) is an operating system that will take care of installing and updating Home Assistant, is managed from the Home Assistant UI, allows creating/restoring snapshots of your configuration, and can easily be extended."
+  description: "[Hass.io](/hassio/) is an full UI managed home automation ecosystem that runs Home Assistant, the Hass.io Supervisor and add-ons. It comes pre-installed on HassOS, but can be installed on any Linux system. It leverages Docker, which is managed by the Hass.io Supervisor."
```

Comment:
Should this be "a"? `is an full`

Reply:
the `an` is not for `full`, but for `UI` (`an UI`, since it an abbreviation).